services/datamanager/builtin/builtin.go

```diff
@@ -679,3 +713,25 @@ func (svc *builtIn) updateDataCaptureConfigs(
 func generateMetadataKey(component, method string) string {
 	return fmt.Sprintf("%s/%s", component, method)
 }
+
+//nolint:unparam
+func pollFilesystem(ctx context.Context, wg *sync.WaitGroup, captureDir string, syncer *datasync.Manager, logger logging.Logger) {
+	t := time.NewTicker(filesystemPollInterval)
```

Comment:
Just wondering for my own knowledge, was lint suggesting a race condition with clock, or did you run into that when testing locally?